src/models/Event.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
+
+const eventSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: [true, "please provide an event name!"],
+    },
+    group: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "Group",
+      unique: true,
+    },
+    creator: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "User",
+      unique: true,
+    },
+    votes: [{ type: mongoose.Schema.Types.ObjectId, ref: "User" }],
```

Comment:
Yeah can wait on this, since the idea of voting is not yet super clear that whether it should for events or activities. I'll take this model out into a separate branch.